packages/itwinui-css/backstop/tests/tabs.html

```diff
@@ -62,7 +62,7 @@ <h2>Default</h2>
                 class="iui-tab-icon"
                 aria-hidden="true"
               ></svg-placeholder>
-              <span class="iui-tab-label">
+              <span class="iui-tab-info">
                 <div>Active</div>
                 <div class="iui-tab-description">Sublabel</div>
               </span>
```

Comment:
Can we explore this markup? 

```html
<button class="iui-tab">
  <svg-x class="iui-tab-icon">...</svg-x>
  <span class="iui-tab-label">The label</span>
  <span class="iui-tab-description">The sublabel</span>
</button>
```

Should be achievable using css grid.
```css
.iui-tab {
  display: grid;
  grid-template-columns: auto minmax(0, 1fr);
  grid-template-areas:
    'icon label'
    'icon description';
}
```

Reply:
I tried and pushed this implementation, but I'm having trouble fixing this overflow issue without the use of a wrapper `div` element for the tab label and description. Any suggestions would be appreciated!

![image](https://github.com/iTwin/iTwinUI/assets/78564221/fdc00c71-84f5-4902-8895-69ebc43bb0cf)

This new DOM structure is present in the second tab of the third example and the first three tabs in the fourth example in the HTML demo pages for now. I didn't want to apply this change to the rest of the examples or React until I'm certain this implementation works.

Reply:
you can try this:
```diff
- grid-template-columns: auto minmax(0, 1fr)
- grid-template-columns: auto minmax(min-content, 1fr)
```

Reply:
> edit: `grid-template-areas` hardcodes everything, so will not work because our icons and descriptions are optional. However, an implicit grid could work if the areas are defined by children.
> 
> ```css
> .iui-tab {
>   display: grid;
>   grid-template-columns: auto minmax(0, 1fr);
> }
> .iui-tab-icon {
>   grid-column: 1;
>   grid-row: 1 / span 2;
> 
>   &:not(:last-child) {
>     margin-inline-end: var(--iui-size-xs);
>   }
> }
> .iui-tab-label {
>   grid-area: 1 / 2;
> }
> .iui-tab-description {
>   grid-area: 2 / 2;
> }
> ```

I did not see this edit until just now! I ended up finding a different CSS solution that works for all cases, but if this solution is better than the one I've pushed please let me know